### DISCUSSION TITLE
Is this still being worked on? or maybe we have a discord?

### DISCUSSION THREAD

**Discussion by Jhazewindus** [tanjeffreyz/auto-maple#163] [General]
Is this still being worked on? or maybe we have a discord?

**Comment by alansete** [comment]
Looks like it isn't being developed anymore, I'd be great to make a discord for the community tho

**Reply by Jhazewindus** [reply]
I actually have it working rn after some tweaking so it definitely works😊 also made some more command books

**Reply by alansete** [reply]
we could start a discord server to share some of the work, I'm also working on it, takes time to tweak things especially with classes that use flash jump, but the code itself is fantastic

**Reply by Jhazewindus** [reply]
https://discord.gg/TWc6JHkH4f

Here is the link :P thought i would set something up
